### DISCUSSION TITLE
Video flashes and goes sometimes just shows dark

### DISCUSSION THREAD

**Discussion by AfterlifeUK** [mifi/lossless-cut#2183] [Support] [answered]
When previewing videos in losslesscut they flicker, i have no other alternative to use since Microsoft photos video turns out washed out and grainy when trimmed, and this software also has alot of problems

I use instant replay to gather videos

codec: H.264/HEVC. with a 4070

https://streamable.com/slopp7 < preview of the issue

**Comment by mifi** [comment]
which version? looks very much like this problem: https://github.com/mifi/lossless-cut/issues/2083#issuecomment-2357067241

**Comment by AfterlifeUK** [comment]
ok i will have a look, thanks, sorry if i posted a duplicate

**Reply by AfterlifeUK** [reply, marked as answer]
![image](https://github.com/user-attachments/assets/dad64017-48d0-473a-819b-47bd32f9c450)


Yup, disabling Multi frame sampled aa seems to have fixed the issue, Thanks for solution
